board-imx7d-pico.yml

```diff
@@ -1,13 +1,11 @@
 header:
   version: 8
+  includes:
+    - kas.yml
```

Comment:
In fact, for the main kas file, I would prefer the include with the commandl-line.
As explained in [1], command line inclusion allows to create configurations on-demand.

So, for example, a user can use its own file if needed:

```
$: kas build meta-swupdate-dev/kas-user.yml:meta-swupdate-dev/board-<machine-name>.yml
```

[1] - https://kas.readthedocs.io/en/1.0/userguide.html#including-configuration-files-via-the-command-line